inst/rmarkdown/templates/rjournal_article/resources/template.tex

```diff
@@ -18,10 +18,19 @@
 
 $for(author)$
 \address{%
-$author.name$\\
-$author.affiliation$\\
-$for(author.address)$$author.address$$sep$\\ $endfor$\\
+$author.name$
+$for(author.affiliation/pairs)$
+  $if(it.value)$
+    $for(it.value.name)$\\$it.value.name$$endfor$%
+    $for(it.value.address)$$it.value.address$$sep$ \\$endfor$\\%
+  $else$
+   \\$author.affiliation$\\
+  $endif$
+$endfor$
+$for(author.address)$$author.address$$sep$ \\$endfor$%
 }
-$if(author.email)$$author.email$$endif$
+$if(author.url)$\\$author.url$$endif$%
+$if(author.orcid)$\\\textit{ORCiD: \href{https://orcid.org/$author.orcid$}{$author.orcid$}}$endif$%
+$if(author.email)$\\$author.email$$endif$
```

Comment:
Per the `RJtemplate.tex` file from RJournal, this should be inside `\adress{}`

From the template tex file of R journal: 
```tex
\address{Author Three\\
  Affiliation\\
  Address\\
  Country\\
  (ORCiD if desired)\\
  \email{author3@work}}
```

* For email, we should also use like the template : `\email`. 

* Url does not seem to be part of the RJournal template. Do you know it this can really be added ?

Reply:
@cderv Thanks for double-checking.

1. I modified the address braces as requested. However, the `\email{}` does not work and causes odd LaTeX errors. Since, however, the e-mail is automatically formated correctly already, I presume that there is a conflict with some LaTeX package that was updated after the R Journal template was created. Because I already get the needed `\href{mailto:...}` tag even without using `\email{}`.

2. Regarding the URL: This is a very good question. I have seen this in published R Journal articles (example: https://journal.r-project.org/archive/2019/RJ-2019-049/RJ-2019-049.pdf). However, as you wrote, it is not part of the official template. Should I remove it? It does not seem to be a problem and in the example, one author is a former editor of the journal, so I guess it is ok. But of course, if you prefer, I remove this.

Reply:
Thanks for the example ! If it already has been published with one, we can assume it can be done. 👍 
Maybe we should add comment in the skeleton to precise which field are optional (`url` and `orcid` from what I understand.)

About `\email`,
* Currently we get `\href{mailto:author1@work}{\nolinkurl{author1@work}}` per pandoc writer. 
* the Rjournal define `\email` like this : 
https://github.com/rstudio/rticles/blob/ec5e82b21c89ecb2d62ea935b1a3104fd0e48375/inst/rmarkdown/templates/rjournal_article/skeleton/RJournal.sty#L231

The later may not work correctly with pandoc without escaping the `@`
```yaml
email:  author1\@work
```

I was just wondering the difference. I guess we can leave as is, as nobody said this wasn't accepted before ?

Reply:
@cderv Ah ok, thanks for the hint. So it is pandoc which acts smart. I thought it is a LaTeX package (e.g., 'url') that parses the e-mail addresses.  Anyway, yes, then we should leave it as it is. The entire idea of the command is to have this `\href{mailto:...}`.

Reply:
Perfect. So this is resolved!